conference-finder/src/App.css

```diff
@@ -20,3 +20,18 @@
 .App-link {
   color: #09d3ac;
 }
+
+.card {
+  width: 100%;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  transition: 0.3s
```

Comment:
What's the transition for? I'm not sure if it's doing anything. Also it's good practice to end all css in a semi colon. Even though it won't break as it is, when you come along to edit the css you might forget that you left it off and wonder why it's erroring

Reply:
To be honest we added it cause it looked nice